### PR TITLE
update data policies with data use agreement

### LINKED ISSUE #97: update data policies page
include links to Data Use Agreement, other docs

### PR DESCRIPTION
**Pull request by wholmgren**
closes #97

### CODE REVIEW

File: datapolicies.md [hunk 1]
```diff
@@ -2,125 +2,91 @@
...
+of an organization before its employees will be allowed to upload data,
+view data contributed by other users, or generate summary statistics.
+
+The data policies can be summarized as:
+
+* Signing the DUA does **not** obligate an organization to upload data,
+  nor does it obligate an organization to share uploaded data.
+* Organizations retain ownership of the data they upload to the framework.
+* Users upload data to the framework on behalf of organizations.
+* Users have complete control over how their organization's data may be
```
Comment (alorenzo175) [inline datapolicies.md]: not sure how to say it, but is admin users that have control

File: datapolicies.md [hunk 2]
```diff
@@ -2,125 +2,91 @@
...
-below. The web interface will show an organization a list of all of the
-submitted metadata/data it owns and the users/organizations that can
-access each data source.
-
-Under some use cases, anonymized time series data and or summary statistics
-derived from the data are owned by the framework. This ensures trial fairness,
-transparency, and reproducibility. The framework is not allowed
-to sell, donate, or otherwise disclose anonymized data or statistics.
+users to access to the data. The web interface shows
+an organization administrator a list of all of the submitted metadata/data
```
Comment (alorenzo175) [inline datapolicies.md]: should organization administrator be defined above?

File: datapolicies.md [hunk 3]
```diff
@@ -2,125 +2,91 @@
...
+it owns and the access roles given to other users. Please see the
+[Data Access Control documentation](/data-access-workflow/) for more
+information.
 
+Under some use cases, anonymized time series data and/or summary
```
Comment (alorenzo175) [inline datapolicies.md]: which cases? too broad. Only during trials?

File: datapolicies.md (hunk 1 above)
Comment (alorenzo175) [reply datapolicies.md]: easier to do if you define admin user above

File: datapolicies.md [hunk 4]
```diff
@@ -2,125 +2,91 @@
...
+
+The data policies can be summarized as:
+
+* Signing the DUA does **not** obligate an organization to upload data,
+  nor does it obligate an organization to share uploaded data.
+* Organizations retain ownership of the data they upload to the framework.
+* Users upload data to the framework on behalf of organizations.
+* Users have complete control over how their organization's data may be
+  accessed by other users.
+* Users may delete their organization's data from the Arbiter at any time.
```
Comment (alorenzo175) [inline datapolicies.md]: admin or user w/ permission, just so people don't think it's willy nilly control within the org

File: datapolicies.md (hunk 3 above)
Comment (alorenzo175) [reply datapolicies.md]: and I guess for reporting things like data volume

**Comment by wholmgren** [comment]
Thanks, good idea to define organization administrator. The text is a little more wonky, but the distinction could be important for some organizations. Now it's (starting to be) a better bridge from the DUA to the documentation.

**Pull request merged**